conf.d/docker_daemon.yaml.example

```diff
@@ -0,0 +1,131 @@
+  # container with host directories mounted on a different folder. Default: /.
+  # Example for the docker-dd-agent container:
+  # docker_root: /host
+
+  # Timeout in seconds for the connection to the docker daemon
+  # Default: 5 seconds
+  #
+  # timeout: 10
+
+  # The version of the API the client will use. Specify 'auto' to use the API version provided by the server.
+  # api_version: auto
+
+  # Use TLS encryption while communicating with the Docker API
+  #
+  # tls: False
+  # tls_client_cert: /path/to/client-cert.pem
+  # tls_client_key: /path/to/client-key.pem
+  # tls_cacert: /path/to/ca.pem
+  # tls_verify: True
+
+instances:
+  - ## Daemon and system configuration
+    ##
+
+    # URL of the Docker daemon socket to reach the Docker API. HTTP/HTTPS also works.
+    # Warning: if that's a non-local daemon, we won't be able to collect performance metrics.
+    #
+    url: "unix://var/run/docker.sock"
+
+    # Cgroup root (HIDDEN OPTION?)
+    # Directory where cgroup files are located, used to generate performance metrics.
+    # Default: None (auto-detected)
+    #
+    # cgroup_path:
+
+    # Cgroup layout (HIDDEN OPTION?)
+    # How cgroup files are organized inside the cgroup root.
+    # Default: None (auto-detected)
+    # Example: "/%(cgroup_root)s/docker/%(container_id)s/%(file)s"
+    #
+    # cgroup_layout: None
+
+
+    ## Data collection
+    ##
+
+    # Create events whenever a container status change.
+    # Defaults to true.
+    #
+    # collect_events: false
+
+    # Collect disk usage per container with docker.container.size_rw and
+    # docker.container.size_rootfs metrics.
+    # Warning: This might take time for Docker daemon to generate,
+    # ensure that `docker ps -a -q` run fast before enabling it.
+    # Defaults to false.
+    #
+    # collect_container_size: false
+
+    # Collect images stats
+    # Number of available active images and intermediate images as gauges.
+    # Defaults to true.
+    #
+    collect_images_stats: false
+
+    # Collect disk usage per image with docker.image.size and docker.image.virtual_size metrics.
+    # The check gets this size with the `docker images` command.
+    # Requires collect_images_stats to be enabled.
+    # Defaults to false.
+    #
+    # collect_image_size: false
+
+
+
+    # Exclude containers based on their tags
+    # An excluded container will be completely ignored. The rule is a regex on the tags.
+    #
+    # How it works: exclude first.
+    # If a tag matches an exclude rule, it won't be included unless it also matches an include rule.
+    # Examples:
+    # exclude all, except ubuntu and debian.
+    # exclude: [".*"]
+    # include: ["docker_image:ubuntu", "docker_image:debian"]
+    #
+    # include all, except ubuntu and debian.
+    # exclude: ["docker_image:ubuntu", "docker_image:debian"]
+    # include: []
+    #
+    # Default: include all
+    # exclude: []
+    # include: []
```

Comment:
I think it's a bit weird that when `exclude = []` it implies exclude **none**, but when `include = []` it implies include **all**. I know that our code actually defaults include to `[]` when it is absent so that it is falsy. But I wonder if the doc needs to leak this implementation detail. Probably just a matter of opinion